build/Dockerfile.tz

```diff
@@ -0,0 +1,4 @@
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
+RUN microdnf clean all
+RUN microdnf update
+RUN microdnf install tzdata-java
```

Comment:
Why `tzdata-java`? I mean the `java` specific in here.

Reply:
no. will remove the java thingy